gulpTasks/strings.js

```diff
@@ -6,7 +6,7 @@ function buildStrings(cb) {
   dict.merge(strings.load('./strings/strings.json', { module: 'Coveo' }));
 
   dict.writeDeclarationFile('./src/strings/Strings.ts');
-  dict.writeDefaultLanguage('./src/strings/DefaultLanguage.ts', 'en', './strings/cultures/globalize.culture.en-US.js', true);
+  dict.writeDefaultLanguage('./src/strings/DefaultLanguage.ts', 'en');
```

Comment:
This change is not strictly needed, but I adjusted writeDefaultLanguage [only supports two arguments](https://github.com/coveo/search-ui/blob/master/strings/strings.js#L110).